numpy/core/fromnumeric.py

```diff
@@ -890,6 +890,14 @@ def argsort(a, axis=-1, kind='quicksort', order=None):
     array([[0, 1],
            [0, 1]])
 
+    Indices of the sorted elements of a N-dimensional array:
+    >>> np.unravel_index(np.argsort(x, axis=None), x.shape)
+    (array([0, 1, 1, 0]), array([0, 0, 1, 1]))
+    >>> from np.testing import assert_equal
+    >>> assert_equal(x[(array([0, 1, 1, 0]), array([0, 0, 1, 1]))], np.sort(x, axis=None))
+    >>> list(zip(*np.unravel_index(np.argsort(x, axis=None), x.shape)))
+    [(0, 0), (1, 0), (1, 1), (0, 1)]
```

Comment:
This might be clearer as:

```
>>> ind = np.unravel_index(np.argsort(x, axis=None), x.shape)
>>> ind
(array([0, 1, 1, 0]), array([0, 0, 1, 1]))
>>> x[ind]  # same as np.sort(x, axis=None)
<whatever it is>
```